frontend/src/main.tsx

```diff
@@ -1,15 +1,12 @@
-import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App";
 import { BrowserRouter as Router } from "react-router-dom";
 import { RecoilRoot } from "recoil";
 
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
-  <React.StrictMode>
```

Comment:
In case we wonder where this went in the future:
This was causing a mysterious double rendering that messed with the "onLeaveRoom" code introduced by this PR.

Reply:
https://stackoverflow.com/questions/49055172/react-component-mounting-twice